app/components/ContestRanking/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { PropTypes } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import Paper from 'material-ui/Paper';
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+
+import messages from './messages';
+
+const comprare = (entry1, entry2) => {
```

Comment:
Komparator jest skrajnie skomplikowany. Wydaje mi się, że jest to coś, do czego powinien być test jednostkowy.